sqlglot/dialects/prql.py

```diff
@@ -53,6 +55,18 @@ class Parser(parser.Parser):
                 _select_all(self._parse_table()), distinct=False, copy=False
             ),
             "SORT": lambda self, query: self._parse_order_by(query),
+            "AGGREGATE": lambda self, query: self._parse_selection(
+                query, self._parse_aggregate, append=False
+            ),
+        }
+
+        AGG_FUNCTIONS = {
+            "MIN": lambda arg: exp.Min(this=arg),
```

Comment:
i'm not sure this is right, we shouldn't have to hardcode all of these, can we programatically do this better? like just parse all the functions without paranethesis and map that to the func?